src/lib.rs

```diff
@@ -93,9 +93,9 @@ fn rustfmt(file_path: &Path) {
     }
 }
 
-pub fn generate_wrappers<T: AsRef<str>>(file_names: &[T], out_dir: &str) {
+pub fn generate_wrappers<T: AsRef<str>>(file_names: &[T], out_dir: &str, protobuf: bool) {
```

Comment:
Rather than using a bool (which is confusing for the caller), could you create a two-valued enum (called something like `ImplMessageTrait`) with variants `Yes` and `No` (or with better names). The effect of this is that when you see the call, it is clearer what is going on: `generate_wrappers(..., false)` vs `generate_wrappers(..., ImplMessageTrait::No)`.